build/Publish.targets

```diff
@@ -71,21 +71,22 @@
 
     <ItemGroup>
       <CliVersionBadgeToUpload>
-        <RelativeBlobPath>$(BlobStoragePartialRelativePath)/$(FullNugetVersion)/$([System.String]::Copy('%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
+        <RelativeBlobPath>$(Product)/$(FullNugetVersion)/$([System.String]::Copy('%(Filename)%(Extension)').Replace('\' ,'/'))</RelativeBlobPath>
+        <ManifestArtifactData>ShipInstaller=dotnetcli</ManifestArtifactData>
       </CliVersionBadgeToUpload>
     </ItemGroup>
 
-    <UploadToAzure
-      AccountKey="$(ArtifactCloudDropAccessToken)"
-      AccountName="$(ArtifactCloudDropAccountName)"
-      ContainerName="$(ArtifactContainerName)"
-      Items="@(CliVersionBadgeToUpload)"
-      Overwrite="$(OverwriteOnPublish)" />
-
-    <SetBlobPropertiesBasedOnFileType
-      AccountKey="$(ArtifactCloudDropAccessToken)"
-      AccountName="$(ArtifactCloudDropAccountName)"
-      ContainerName="$(ArtifactContainerName)"
-      Items="@(CliVersionBadgeToUpload)" />
+    <Message Text="Publish badges to blob feed started: $(ArtifactCloudDropURL)" />
+    <PushToBlobFeed ExpectedFeedUrl="$(ArtifactCloudDropURL)"
+                    AccountKey="$(ArtifactCloudDropAccessToken)"
+                    ItemsToPush="@(CliVersionBadgeToUpload)"
+                    ManifestBranch="$(BranchName)"
+                    ManifestBuildId="$(FullNugetVersion)"
+                    ManifestCommit="$(CommitHash)"
+                    ManifestName="$(BuildName)"
+                    Overwrite="false" 
+                    PublishFlatContainer="true" 
+                    SkipCreateManifest="$(IsNotOrchestratedPublish)" />
```

Comment:
I think this might have an issue.  Unless I missed a place where it's referenced, IsNotOrchestratedPublish isn't set to 'false' when the PB_PublishBlobFeedUrl is set, so this input would have an empty value.  I don't know what the msbuild semantics are in this case, but it's probably good practice to set IsNotOrchestratedPublish explicitly to false if PB_PublishBlobFeedUrl is non-empty.

Reply:
If we are explicitly setting "IsNotOrchestratedPublish"; then we can go back to the more-understandable positive case. Changed.

Reply:
Done...